modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/ContactInfo/ContactInfoForm.tsx

```diff
@@ -575,13 +569,15 @@ const ContactInfo = () => {
 								setHasError({
 									...hasError,
 									state:
-										stateValue === '1' || stateValue === '2'
+										stateValue === '' ||
+										stateValue === 'CHOOSE AN OPTION'
 											? true
 											: false,
 								});
 							}}
+							value={form.state}
 						>
-							{stateOptions.map((item) => (
+							{CONSTANTS.STATES_US.map((item) => (
```

Comment:
if it makes sense to you, change this data
```suggestion
							{CONSTANTS.US_STATES.map((us_state) => (
```

Reply:
Done!